src/parsers/fluent.js

```diff
@@ -19,29 +23,33 @@ export default class FluentParser {
   }
 
   parse() {
-    const [entries, errors] = parseFluent(this._sourceString);
+    const parser = new FluentSyntaxParser();
+    const resource = parser.parse(this._sourceString);
+
     this.parsedData = {};
 
-    Object.keys(entries).forEach((id) => {
-      this.parsedData[id] = entries[id];
-    });
+    resource.body.forEach((entry) => {
+      if (entry.type === 'Junk') {
```

Comment:
If `fluent-syntax` provides a constant for this it'd be nice, magic constants just make me scared.

Reply:
Unfortunately not, I might push a follow-up PR to clean this up a bit and think about better tests.

Reply:
It provides a type you can test against `instanceof` - https://github.com/projectfluent/fluent.js/blob/master/fluent-syntax/src/ast.js#L240

Reply:
:+1: on importing the `Junk` class and turning this into `entry instanceof Junk` in a follow up as @zbraniecki pointed out.